### PR TITLE
Programmatically generate default WKWebview object for PassedIn mode on MSAL MacOS test app. 

### PR DESCRIPTION
**Pull request by NerevarineRule**
## Proposed changes

This work was part of this PR -  https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/1103
But somehow the PR 1103 accidentally got merged when the other PR was merged. (https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/1104)
I am not sure how that happened.

The purpose of this PR is to programmatically initialize a default WKWebview object in the MSAL MacOS test app and then use default WKWebview configuration setting returned from a new public API from the PR 1103. 

This Webview object is in turn used by MSAL MacOS test app when the user selects "PassedIn" mode. 

By doing so, we adhere to the default config setting that we advertise to the developers and also verify that it works properly.

Currently, the default WKWebview object is created as a weak IBOutlet object from the storyboard, rendering it impossible to use custom configuration settings. 

## Type of change

- [ ] Feature work
- [x] Bug fix
- [ ] Documentation
- [ ] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

## Risk

- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [x] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [ ] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

### CODE REVIEW

File: MSAL/src/public/MSALDefinitions.h [hunk 1]
```diff
@@ -83,12 +83,7 @@ typedef NS_ENUM(NSInteger, MSALWebviewType)
     MSALWebviewTypeSafariViewController,
     
 #endif
-    /** Use WKWebView
-     It is recommended to use the default webview configuration setting provided by a public MSAL API.
-     ex:
-     WKWebViewConfiguration *defaultWKWebConfig = [MSALWebviewParameters defaultWKWebviewConfiguration];
-     WKWebView *embeddedWebview = [[WKWebView alloc] initWithFrame:yourWebview.frame configuration:defaultWKWebConfig];
-    */
+    /** Use WKWebView */
```
Comment (kaisong1990) [inline MSAL/src/public/MSALDefinitions.h:86]: 👍

File: MSAL/test/app/mac/MSALAcquireTokenViewController.m [hunk 2]
```diff
@@ -56,11 +56,10 @@ @interface MSALAcquireTokenViewController ()
 @property (weak) IBOutlet NSSegmentedControl *webViewSegment;
 @property (weak) IBOutlet NSSegmentedControl *validateAuthoritySegment;
 @property (weak) IBOutlet NSStackView *acquireTokenView;
-@property (weak) IBOutlet WKWebView *webView;
 @property (weak) IBOutlet NSPopUpButton *userPopup;
 @property (weak) IBOutlet NSSegmentedControl *authSchemeSegment;
 
-
+@property (nonatomic) WKWebView *webView;
```
Comment (kaisong1990) [inline MSAL/test/app/mac/MSALAcquireTokenViewController.m:62]: Any reason to make as ```nonatomic```

**Review by kaisong1990 — APPROVED** [review]
LGTM with a minor question

File: MSAL/test/app/mac/MSALAcquireTokenViewController.m (hunk 2 above)
Comment (NerevarineRule) [reply MSAL/test/app/mac/MSALAcquireTokenViewController.m:62]: Good catch. I guess there's no reason to make this nonatomic, as we want to make sure any operation on the webView object is thread-safe. it's atomic by default, and we want thread operation to be serialized on this object.

**Pull request merged**